script.js

```diff
@@ -10,3 +10,36 @@ document.querySelectorAll('.nav-item').forEach((n) => n.addEventListener('click'
   hamburger.classList.remove('active');
   navList.classList.remove('active');
 }));
+
+// Preserve data in the browser
+function fillInputs() {
+  // seting on load
+  const username = document.getElementById('text1n');
+  const valueForName = localStorage.getItem('username');
+  if (valueForName) {
+    username.value = valueForName;
+  }
+  username.addEventListener('change', (event) => {
+    username.value = event.target.value;
+    localStorage.setItem('username', event.target.value);
+  });
+  const email = document.getElementById('text2');
+  const ValueFormEmail = localStorage.getItem('email');
+  if (ValueFormEmail) {
+    email.value = ValueFormEmail;
+  }
+  email.addEventListener('change', (event) => {
+    email.value = event.target.value;
+    localStorage.setItem('email', event.target.value);
+  });
+  const text = document.getElementById('text3t');
+  const valueForText = localStorage.getItem('text');
+  if (valueForText) {
+    text.value = valueForText;
+  }
+  text.addEventListener('change', (event) => {
+    text.value = event.target.value;
+    localStorage.setItem('text', event.target.value);
+  });
```

Comment:
- [ ] Good job so far 👏. Data is persistent in the browser but among the project requirements, you are expected to store form data for all inputs in one single object and then save that single object into local storage 😃 👍 
The object should look like the following 👇 
```
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};

```

Please refactor your codes in order to store all your form inputs data in a single object and then store and retrieve it from Local storage 👍 
⚠️ Make sure that the functionality is still working well 😉